### DISCUSSION TITLE
Help with MODBUS connection

### DISCUSSION THREAD

**Discussion by andyfishes** [artisan-roaster-scope/artisan#1299] [General]
Hello fellow roasters, I need help with connecting my roaster to artisan. It shows MODBUS connected but then the values are u.u and it also shows an error. My old computer (Mackbook Air intel) works fine, but I'm using my new macbook pro M1 and it's showing this error. I'm using an RS485 and I've already tried downloaded the CH340 driver.

Thanks if anyone could just help me find a solution.

<img width="658" alt="Screenshot 2023-11-16 at 10 02 00 PM" src="https://github.com/artisan-roaster-scope/artisan/assets/151051134/cdb4ae71-d171-44eb-a2ea-6dada5de5b04">

**Comment by MAKOMO** [comment]
The formula you typed under "BT Y(x)" in the "Symb ET/BT" tab (tab 3 of the Device assignment dialog; menu `Config >> Device`) has a syntax error. Fix that typo or remove that formula completely and put an empty string there.

**Reply by andyfishes** [reply]
It works now. Many thanks!
